js/showpassword.js

```diff
@@ -0,0 +1,38 @@
+// for login
+function box2()
+{
+ 
+  var password=document.getElementById("pswd");
+  var x=document.getElementById("box").checked;
+  if(x==true)
+  {
+    password.type="text";
+  }
+  else
+  {
+    password.type="password";
+    
+  }
+}
+
+
+// for sign up
+function box1()
+{
+  
+  var password=document.getElementById("pswd");
```

Comment:
did you try combining `box1()` and `box2()` functions using the `regex` with `.getElementById` as mention earlier?

Reply:
@siv2r I am finding difficulty in regex.
but now I had combined those 2 functions into 1 by adding some if else conditions .
I hope its fine

Reply:
Hey, the point is not to have a single function. It is to have a concise, readable function. This seems unnecessarily long. We would be better of with two separate functions instead of a single one with many `if` `else` statements.

Reply:
It is good that you find `regex` difficult since you are learning something new, which is the whole point of contributing to opensource. If you have some doubts, you can ask in discord or the issues thread on GitHub. Take your time to get hold of `regex` and try its implementation. You are not required to solve this issue immediately, go on your own pace :)